### ISSUE TITLE
Even .cal-hour elements show body background

### ISSUE THREAD

**Issue by l0wlik34G6** [mattlewis92/angular-calendar#951]
**Describe the bug**

If you have a background color set on your container, which holds the mwl-calendar-day-view, a line is drawn on the right side of even cal-hour elements.
To avoid this, I changed the angular-calendar.css as follows:

```
   .cal-day-view .cal-hour:nth-child(odd) `
     background-color: #fafafa; }
+  .cal-day-view .cal-hour:nth-child(even) {
+    background-color: #fff; }
   .cal-day-view .cal-hour:not(:last-child) .cal-hour-segment,
   .cal-day-view .cal-hour:last-child :not(:last-child) .cal-hour-segment {
```

I don't know if this is the right way to do so, but for me it woked :)

**Minimal reproduction of the problem with instructions**

For reproduction, just apply a color (except white and #fafafa of course) to your e.g. body and add the mwl-calendar-day-view component to your app component.
```
body {
    color: #000;
}
```

**Screenshots**

![image](https://user-images.githubusercontent.com/48749340/56160586-d77dfe80-5fc7-11e9-9e91-6b6a7b305e41.png)

**Versions**

- `@angular/core`: 7.2.13
- `angular-calendar`: 0.27.3
- Browser name and version: chrome 73.0

**Comment by mattlewis92** [comment]
Hey, thanks for opening an issue, I tried to reproduce but couldn't, are you able to make a runnable stackblitz that I can debug further that shows your problem? Thanks!
<img width="1133" alt="Screenshot 2019-04-16 at 14 54 12" src="https://user-images.githubusercontent.com/6425649/56188230-861d5000-6057-11e9-9d58-9b25101ce268.png">

**Comment by l0wlik34G6** [comment]
Ye Sure: https://stackblitz.com/edit/angular-calendar-951

**Comment by mattlewis92** [comment]
Ahh I see now, it's because the demos use bootstrap which is fixing whatever issue is causing the problem, thanks! 😄

**Comment by mattlewis92** [comment]
Should be fixed now in `0.27.4`, thanks for reporting and your help reproducing! 😄
